pkg/codegen/schema/loader.go

```diff
@@ -32,6 +36,20 @@ type pluginLoader struct {
 
 	host    plugin.Host
 	entries map[string]PackageReference
+	files   []*os.File
+	mmaps   []mmap.MMap
+
+	cacheOptions PluginLoaderCacheOptions
+}
+
+// Caching options intended for benchmarking or debugging:
+type PluginLoaderCacheOptions struct {
```

Comment:
How are these options set? All of the fields are unexported.

Reply:
A benchmark test in the same package.

I'll make this type and the `NewPluginLoaderWithOptions` constructor private to the package.